cmd/helm/plugin_test.go

```diff
@@ -86,11 +86,13 @@ func TestLoadPlugins(t *testing.T) {
 		long   string
 		expect string
 		args   []string
+		code   int
 	}{
-		{"args", "echo args", "This echos args", "-a -b -c\n", []string{"-a", "-b", "-c"}},
-		{"echo", "echo stuff", "This echos stuff", "hello\n", []string{}},
-		{"env", "env stuff", "show the env", "env\n", []string{}},
-		{"fullenv", "show env vars", "show all env vars", envs + "\n", []string{}},
+		{"args", "echo args", "This echos args", "-a -b -c\n", []string{"-a", "-b", "-c"}, 0},
+		{"echo", "echo stuff", "This echos stuff", "hello\n", []string{}, 0},
+		{"env", "env stuff", "show the env", "env\n", []string{}, 0},
+		{"exitwith", "exitwith code", "This exits with the specified exit code", "", []string{"2"}, 2},
```

Comment:
This doesn't actually verify which exit code is passed to `os.Exit`. But this does ensure that we don't break the helm plugin system to not swallow non-zero exit codes other than 1.

Reply:
I mean this compliments `TestPluginExitCode` rather than duplicating it.